config/opal_check_libnl.m4

```diff
@@ -0,0 +1,230 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2015-2016 Research Organization for Information Science
+dnl                         and Technology (RIST). All rights reserved.
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+dnl
+dnl Helper function to parse --with-libnl* options
+dnl
+dnl $1: variable name
+dnl $2: --with-<foo> value
+dnl
+AC_DEFUN([OPAL_LIBNL_PARSE_WITH],[
+	case "$2" in
+	no)
+		# Nope, don't want it
+		opal_want_$1=no
+		;;
+	yes)
+		# Yes, definitely want it
+		opal_want_$1=yes
+		;;
+	default)
+		# Default case -- try and see if we can find it
+		opal_want_$1=default
+		opal_$1_location=/usr
```

Comment:
this value is never used this way. it is passed to ```OPAL_CHECK_PACKAGE``` in which ```/usr``` or ```/usr/local``` is kind of removed